packages/theming/css-vars-usage.json

```diff
@@ -0,0 +1,419 @@
+[
```

Comment:
Do we need this file? It looks like something automatically added.

Reply:
The file is now in the master branch (previously someone deleted it accidentally). Basically if @nnaydenow  merges the PR with the master, it will disappear from files changed view. The file is a report of the CSS vars that are used across all components.

Reply:
As ilhan said the file was accidentally deleted. Now, I've rebased to main branch and the file is not part of the commit.